corehq/apps/app_manager/app_schemas/tests/test_app_diffs.py

```diff
@@ -245,7 +246,8 @@ def tearDown(self):
     def test_remove_save_property(self):
         self._add_question(self.app1.modules[0].forms[0])
         self._add_question(self.app2.modules[0].forms[0])
-        self.factory1.form_requires_case(self.app1.modules[0].forms[0], update={'foo': '/data/name'})
+        self.factory1.form_requires_case(self.app1.modules[0].forms[0],
```

Comment:
Are there instance where this is called with ConditionalCaseUpdate classes that have non-default update_mode (I don't think I saw any in this PR)?

I'm just wondering if it wouldn't be better to leave this API as it is an convert the path values into ConditionalCaseUpdate classes in the method. Tests can always fall back to setting up the case management directly if necessary.

```python
def form_requires_case(...., update) {
    update = {name: ConditionalCaseUpdate(question_path=path) for name, path in update.items()}
```

Reply:
I don't think there is such an instance. I agree it'd keep things cleaner, but making this change would prevent the ability to add an `update_mode` when using this method in the future if that's something we wanted to do.